CHANGELOG.md

```diff
@@ -2,6 +2,52 @@
 
 ## v3.7.0
 
+This release introduces group mentions (e.g. `@engineering`) across all packages
+and first-class support for tenants. Learn more about [group mentions](#) and
```

Comment:
Reminder to not forget to link these to the actual doc URLs.

Reply:
Waiting on https://github.com/liveblocks/liveblocks/pull/2655.